.github/workflows/nodejs.yml

```
@@ -0,0 +1,32 @@
+name: Node CI
```

```suggestion
name: Node
```

Since this is visible in the ci list at the bottom, extra unneeded text ;)